src/config/data.js

```diff
@@ -8,7 +8,7 @@ import {
 } from './images'
 
 export const LONDON = 'London'
-export const LISBON = 'Lisbon dist.'
+export const LISBON = 'Lisbon'
```

Comment:
@shakersinthedark Torres Vedras and Parede is not Lisbon, but they are all in the Lisbon district https://en.wikipedia.org/wiki/Lisbon_District. Saying the bootcamp is in Lisbon distr. is true, saying the bootcamp in Lisbon is not true. That can be misleading.

Reply:
Fair enough, but I don't think including '_dist._' is clear enough to be understandable and could potentially hurt the UX more at the beginning of the sales funnel. 

A heading should be immediately understandable and '_dist._' potentially hinders that when a user first goes on the page... I have to admit, the first time I saw it, I didn't know what it meant and it - to me - feels unnecessary from a UX standpoint (i don't have solid evidence either way just FYI). If they're interested in the course they get to the instance page which gives you the granular details of where exactly it is. 

Imho, it's too much cognitive load for someone to try and understand what '_dist._' means in the title of a page and isn't worth the payoff of knowing that it's in a town outside of Lisbon this earlier in the sales funnel. Thing is, if we're trying to attract more people from outside Lisbon, we need the general area to be obvious (or, put another way, the airport a user has to fly to), for their immediate needs.

I removed it cos I feel it's unnecessary and potentially confusing but, when it comes down to it, the curriculum is what people buy - not the location - and we're covered on that so I'm happy to go with whatever either way.

Reply:
@alexlbr cool, will do. I think if we get complaints or bounce rate goes up after this merge, we should discuss it further